src/benchmarks/micro/libraries/System.Runtime.Extensions/Perf.Path.cs

```diff
@@ -59,5 +59,12 @@ public class Perf_Path
 
         [Benchmark]
         public bool IsPathRooted() => Path.IsPathRooted(_testPath);
+
+        [Benchmark]
+        [Arguments(@"C:\..\.\.\folder1\\toremove\.\..\.\folder2\toremove\toremove\..\.\.\..\folder3\.\....\..\.")]
```

Comment:
What is the result of sending a Windows-specific path to `Path.GetFullPath` while running on `Unix`? Does it work anyway, throws or returns null?

Reply:
Based on [this comment](https://github.com/dotnet/performance/pull/1394#issuecomment-657706833), I decided to remove this code.